dpkgfiles/make_deb_package.README

```diff
@@ -0,0 +1,48 @@
+Instructions for using make_deb_package.py outside of Gitian
+
+
+1) We use cowbuilder to build in a chroot, so you need to prepare a chroot with
+   cowbuilder first (instructions from the Debian reproducible build project):
+
+      sudo cowbuilder --create --distribution codename --basepath /var/cache/pbuilder/base-reproducible.cow
```

Comment:
When using Ubuntu Vivid (15.04) as the host (and probably earlier versions, judging by [this question](http://askubuntu.com/questions/174378/cowbuilder-create-distribution-lucid-fails), --components "main universe" (quotation marks necessary) is necessary. Here's what I used. Can you confirm that this doesn't break anything on Debian?

sudo cowbuilder --create --distribution codename --components "main universe" --basepath /var/cache/pbuilder/base-reproducible.cow